riscv-asm.md

```diff
@@ -164,7 +164,7 @@ Directive    | Arguments                      | Description
 .macro       | name arg1 [, argn]             | begin macro definition \argname to substitute
 .endm        |                                | end macro definition
 .type        | symbol, @function              | accepted for source compatibility
-.option      | {rvc,norvc,pic,nopic,push,pop} | RISC-V options
+.option      | {rvc,norvc,pic,nopic,relax,norelax,push,pop} | RISC-V options, more detailed description in [.option](#.option).
```

Comment:
Reformat table, to realign the `|`s?

Nit: "RISC-V options. Refer to [.option](#.option) for a more detailed description."

Reply:
I would prefer don't realign this time since we might need to realign for `.option arch,` soon, my thought is we can did that when converting the format to AsciiDoc.